examples/nodejs-express-launchdarkly/README.md

```diff
@@ -0,0 +1,104 @@
+<a href="https://arcjet.com" target="_arcjet-home">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://arcjet.com/logo/arcjet-dark-lockup-voyage-horizontal.svg">
+    <img src="https://arcjet.com/logo/arcjet-light-lockup-voyage-horizontal.svg" alt="Arcjet Logo" height="128" width="auto">
+  </picture>
+</a>
+
+# Arcjet Dynamic Configuration via LaunchDarkly Feature Flags (with Express for Node.js)
```

Comment:
```suggestion
# Arcjet Dynamic Configuration via LaunchDarkly Feature Flags
```
It makes the permalink worse and doesn't seem worth mentioning when it is already in the description below.